install.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+# check Ubuntu version
+source /etc/os-release
+
+if [[ $UBUNTU_CODENAME != 'jammy' ]]
+then
+    echo "Ubuntu 22.04.1 LTS (Jammy Jellyfish) is required"
+    echo "You are using $VERSION"
+    exit 1
+fi
+
+### Get directory where this script is installed
+BASEDIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+### Install ROS2
+cd ~
+git clone https://github.com/Tiryoh/ros2_setup_scripts_ubuntu.git
+./ros2_setup_scripts_ubuntu/run.sh
+
+source /opt/ros/humble/setup.bash
+mkdir -p ~/mini_pupper_ws/src
+cd ~/mini_pupper_ws/src
+#TODO change URL after testing
+[[ -d ~/mini_pupper_ros_bsp ]] || git clone https://github.com/mangdangroboticsclub/mini_pupper_ros_bsp,git
+
+cd ~/mini_pupper_ws
+rosdep update && rosdep install --from-path src --ignore-src -y --skip-keys microxrcedds_agent --skip-keys micro_ros_agent
+sudo pip install setuptools==58.2.0 # suppress colcon build warning
+colcon build --executor sequential --symlink-install
+
+# Install servo_interface
+cd ~/mini_pupper_ros_bsp/services
+sudo ln -s $(realpath .)/servo_interface.service /etc/systemd/system/
+
+# Install display_interface
+cd ~/mini_pupper_ros_bsp/services
+sudo ln -s $(realpath .)/display_interface.service /etc/systemd/system/
+
+# Install Joystick
+cd ~/mini_pupper_ros_bsp/services
+sudo ln -s $(realpath .)/joystick.service /etc/systemd/system/
+sudo ln -s $(realpath .)/joy_node.service /etc/systemd/system/
+sudo ln -s $(realpath .)/restart_joy.service /etc/systemd/system/
+
+# Install Lidar
+source /opt/ros/humble/setup.bash
+mkdir -p ~/lidar_ws/src
+cd ~/lidar_ws
```

Comment:
How about using `~/ros2_ws/src`  to match the official tutorial? Do we need to separate the workspace?

https://docs.ros.org/en/humble/Tutorials/Beginner-Client-Libraries/Creating-A-Workspace/Creating-A-Workspace.html#create-a-new-directory

Reply:
OK, will do